docs/jaeger-support.asciidoc

```diff
@@ -53,6 +53,34 @@ Based on the endpoint enabled in the previous step, configure the relevant host
 * `apm-server.jaeger.grpc.host` defaults to `localhost:14250`.
 * `apm-server.jaeger.http.host` defaults to `localhost:14268`.
 
+[[jaeger-configure-sampling]]
+*Configure Sampling*
+
+The gRPC endpoint supports probabilistic sampling, which can be used to reduce the amount of data that your agents collect and send.
+Probabilistic sampling makes a random sampling decision based on the configured sampling value.
+For example, a value of `.2` means that 20% of traces will be sampled.
+
+By default, when using Jaeger over gRPC, two things happen:
+
+* APM Server automatically enables the sampling endpoint when `apm-server.jaeger.grpc.enabled` is set to `true`.
+* Jaeger agents poll APM Server for the configured sampling rate.
```

Comment:
I just realized we miss a reference to the [Kibana config](https://www.elastic.co/guide/en/apm/server/master/setup-kibana-endpoint.html). The sampling endpoint is enabled by default, but Kibana is not and needs to be configured.  
Looking at the current Kibana config docs I think they should also be slightly changed to reflect the Jaeger use case.

Reply:
Great catch! I've updated the docs to add a reference to the Kibana config, and to better differentiate between the two different methods for configuring Jaeger sampling.

I'm hesitant to update the Kibana config docs to reference Jaeger though. To me, "central jaeger sampling" is simply a feature of central config. So we should set the tone in the Jaeger docs that "central sampling" is powered by APM Agent configuration. If we do this, the Kibana config documentation doesn't need to mention the Jaeger use case.

I tried to do this in the recent commit. Take a look and let me know what you think